tests/console/yast2_dns_server.pm

```diff
@@ -88,7 +88,6 @@ sub run {
     if (is_sle('<15') || is_leap('<15.1')) {
         assert_screen([qw(yast2-dns-server-step3 yast2_still_susefirewall2)], 90);
         if (match_has_tag 'yast2_still_susefirewall2') {
-            record_soft_failure "bsc#1059569";
```

Comment:
You deleted the record_soft_failure-calls but not the workaround code. The record_soft_failure serves as an explanation why we have this workaround code snippets. If you delete just that then new readers of the code will not know why it is there. In the best case we can delete the "if (match_has_tag…)"-parts, otherwise at least we should keep the bug-refs as a comment explaining why we have that code even though we do not record a soft_fail. I noted this down in https://progress.opensuse.org/issues/40907#note-8 as well so we have this tracked until the ticket is resolved. No need to remember the comment in this PR :)